src/libraries/sendtohelix.proj

```diff
@@ -144,7 +144,22 @@
     <_RuntimeInputs Condition=" '$(Scenarios)' != '' and '$(TargetsWindows)' != 'true' " Include="$(TestHostRootPath)**/*.sh" />
   </ItemGroup>
 
+  <Target Name="IncludeDumpDocsInTesthost"
```

Comment:
Should this also be added for CoreCLR?

Reply:
CoreCLR uses another sendtohelix project. I can hook it up there. However, the CoreCLR payloads don't contain symbols.. so those instructions are going to be a little bit different.

Reply:
Will do this as part of a follow up PR.